src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -410,8 +411,9 @@ export class MdAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
 
     // If it's used in a Material container, we should set it through
     // the property so it can go through the change detection.
-    if (this._inputContainer) {
-      this._inputContainer._mdInputChild.value = inputValue;
+    if (this._inputContainer &&
+        this._inputContainer._control instanceof MdInput) {
```

Comment:
Hmm now that I think about it, its probably better to make the `value` property part of the `MdFormFieldControl` interface so this doesn't have to be hardcoded for `MdInput`. I was only thinking about`mdInput` and `md-select` when I wrote this, but its reasonable to want autocomplete to work with a user's custom input control as well